### ISSUE TITLE
Generating models in custom folder(namespace) with custom base class

### ISSUE THREAD

**Issue by Sayorus** [laravel-idea/plugin#53]
Hello! Can you explain, what i do wrong?
I need generate model in custom place (\App\Models namespace) with custom abstract base class (placed at \App\Models\AbstractModel). All this data I enter to model generator properties. But after generating i see right place but wrong base class(default Illuminate\Database\Eloquent\Model) or wrong place (file placed in app directory) but right base class.

**Comment by adelf** [comment]
Hi. I tried this:

![image](https://user-images.githubusercontent.com/2818394/82127046-78398180-97b9-11ea-9422-eb86b788cc29.png)

And then created SomeModel class. It created in app/Models folder with:
```
<?php

namespace App\Models;

class SomeModel extends AbstractModel
{
}
```

Looks like everyhting is ok...

**Comment by Sayorus** [comment]
Okay, I tried this too:
<img width="1109" alt="изображение" src="https://user-images.githubusercontent.com/7129562/82127818-27c52280-97bf-11ea-97ea-6a4b51835f07.png">
As you see, preferences applied and saved.
Next I open by hotkeys modal window where i choose "Create Model", and enter model name.
Result:
<img width="1631" alt="изображение" src="https://user-images.githubusercontent.com/7129562/82127927-ad48d280-97bf-11ea-923a-224102c9b76b.png">
(On second image preferences window is only open, not edited)

**Comment by adelf** [comment]
I could reproduce some strange behaviour in my local environment. Will try to fix it in the next version (I hope to release it in Monday).

**Comment by adelf** [comment]
Fixed. Thank you so much for reporting this!

**Comment by adelf** [comment]
fixed in 3.0 version. please check and comment if there are some issues.
